kraken-app/src/Components/MarketplacePreview/MarketplacePreview.jsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import CssBaseline from '@mui/material/CssBaseline';
+import Box from '@mui/material/Box';
+import Container from '@mui/material/Container';
+import Pagination from '@mui/material/Pagination';
+import Stack from '@mui/material/Stack';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import NFTCard from '../NFTCard/NFTCard.jsx';
+
+const cards = [1, 2, 3];
+
+function MarketplacePreview(props) {
+
+  const [page, setPage] = React.useState(1);
+  const handleChange = (event, value) => {
+    setPage(value);
+  };
+
+  return (
+    <>
+      <CssBaseline />
```

Comment:
What does the CssBaseline tag do?

Reply:
MUI provides a CssBaseline component to kickstart an elegant, consistent, and simple baseline to build upon.